Caused by: java.lang.NoClassDefFoundError: com/google/api/client/json/jackson2/JacksonFactory

Hi, I've got an exception, after updating AGP to 4.2.1, Gradle 6.7.1, artifactory plugin: 2.1.1

```
Caused by: java.lang.NoClassDefFoundError: com/google/api/client/json/jackson2/JacksonFactory
        at com.google.cloud.artifactregistry.auth.DefaultCredentialProvider.getCredential(DefaultCredentialProvider.java:49)
        at com.google.cloud.artifactregistry.gradle.plugin.ArtifactRegistryGradlePlugin.apply(ArtifactRegistryGradlePlugin.java:84)

```

--- a/artifactregistry-auth-common/build.gradle
+++ b/artifactregistry-auth-common/build.gradle
@@ -1,6 +1,6 @@
 dependencies {
     compile group: 'com.google.http-client', name: 'google-http-client', version:'1.39.2'
-    compile group: 'com.google.auth', name: 'google-auth-library-oauth2-http', version:'1.0.0'
+    compile group: 'com.google.auth', name: 'google-auth-library-oauth2-http', version:'0.27.0'
     compile group: 'org.slf4j', name: 'slf4j-api', version:'1.7.30'
     testCompile group: 'junit', name: 'junit', version: '4.12'
 }

--- a/artifactregistry-auth-common/build.gradle
+++ b/artifactregistry-auth-common/build.gradle
@@ -1,6 +1,6 @@
 dependencies {
-    compile group: 'com.google.http-client', name: 'google-http-client', version:'1.30.2'
-    compile group: 'com.google.auth', name: 'google-auth-library-oauth2-http', version:'0.16.2'
+    compile group: 'com.google.http-client', name: 'google-http-client', version:'1.39.2'
+    compile group: 'com.google.auth', name: 'google-auth-library-oauth2-http', version:'1.0.0'
     compile group: 'org.slf4j', name: 'slf4j-api', version:'1.7.30'
     testCompile group: 'junit', name: 'junit', version: '4.12'
 }

--- a/artifactregistry-auth-common/build.gradle
+++ b/artifactregistry-auth-common/build.gradle
@@ -1,6 +1,7 @@
 dependencies {
     compile group: 'com.google.http-client', name: 'google-http-client', version:'1.39.2'
-    compile group: 'com.google.auth', name: 'google-auth-library-oauth2-http', version:'0.27.0'
+    compile group: 'com.google.http-client', name: 'google-http-client-jackson2', version:'1.39.2'
+    compile group: 'com.google.auth', name: 'google-auth-library-oauth2-http', version:'1.0.0'
     compile group: 'org.slf4j', name: 'slf4j-api', version:'1.7.30'
     testCompile group: 'junit', name: 'junit', version: '4.12'
 }

--- a/artifactregistry-gradle-plugin/build.gradle
+++ b/artifactregistry-gradle-plugin/build.gradle
@@ -5,7 +5,7 @@ plugins {
 
 dependencies {
 	compile gradleApi()
-	compile group: 'com.google.auth', name: 'google-auth-library-oauth2-http', version: '0.16.2'
+	compile group: 'com.google.auth', name: 'google-auth-library-oauth2-http', version: '1.0.0'
 }
 
 pluginBundle {

--- a/artifactregistry-gradle-plugin/build.gradle
+++ b/artifactregistry-gradle-plugin/build.gradle
@@ -5,7 +5,7 @@ plugins {
 
 dependencies {
 	compile gradleApi()
-	compile group: 'com.google.auth', name: 'google-auth-library-oauth2-http', version: '0.27.0'
+	compile group: 'com.google.auth', name: 'google-auth-library-oauth2-http', version: '1.0.0'
 }
 
 pluginBundle {

--- a/artifactregistry-gradle-plugin/build.gradle
+++ b/artifactregistry-gradle-plugin/build.gradle
@@ -5,7 +5,7 @@ plugins {
 
 dependencies {
 	compile gradleApi()
-	compile group: 'com.google.auth', name: 'google-auth-library-oauth2-http', version: '1.0.0'
+	compile group: 'com.google.auth', name: 'google-auth-library-oauth2-http', version: '0.27.0'
 }
 
 pluginBundle {

--- a/artifactregistry-maven-wagon/build.gradle
+++ b/artifactregistry-maven-wagon/build.gradle
@@ -1,8 +1,8 @@
 dependencies {
     compile group: 'org.apache.maven.wagon', name: 'wagon-http-shared', version: '3.3.3'
     compile group: 'org.apache.maven', name: 'maven-plugin-api', version: '3.6.1'
-    compile group: 'com.google.http-client', name: 'google-http-client', version: '1.39.2'
-    compile group: 'com.google.auth', name: 'google-auth-library-oauth2-http', version: '0.27.0'
+    compile group: 'com.google.http-client', name: 'google-http-client', version:'1.39.2'
+    compile group: 'com.google.auth', name: 'google-auth-library-oauth2-http', version:'1.0.0'
     testCompile group: 'org.apache.maven.wagon', name: 'wagon-provider-test', version: '3.3.3'
     testCompile group: 'junit', name: 'junit', version: '4.12'
     compileOnly group: 'org.apache.maven.wagon', name: 'wagon-provider-api', version: '3.3.3'

--- a/artifactregistry-maven-wagon/build.gradle
+++ b/artifactregistry-maven-wagon/build.gradle
@@ -2,7 +2,7 @@ dependencies {
     compile group: 'org.apache.maven.wagon', name: 'wagon-http-shared', version: '3.3.3'
     compile group: 'org.apache.maven', name: 'maven-plugin-api', version: '3.6.1'
     compile group: 'com.google.http-client', name: 'google-http-client', version: '1.39.2'
-    compile group: 'com.google.auth', name: 'google-auth-library-oauth2-http', version: '1.0.0'
+    compile group: 'com.google.auth', name: 'google-auth-library-oauth2-http', version: '0.27.0'
     testCompile group: 'org.apache.maven.wagon', name: 'wagon-provider-test', version: '3.3.3'
     testCompile group: 'junit', name: 'junit', version: '4.12'
     compileOnly group: 'org.apache.maven.wagon', name: 'wagon-provider-api', version: '3.3.3'

--- a/artifactregistry-maven-wagon/build.gradle
+++ b/artifactregistry-maven-wagon/build.gradle
@@ -1,8 +1,8 @@
 dependencies {
     compile group: 'org.apache.maven.wagon', name: 'wagon-http-shared', version: '3.3.3'
     compile group: 'org.apache.maven', name: 'maven-plugin-api', version: '3.6.1'
-    compile group: 'com.google.http-client', name: 'google-http-client', version: '1.30.2'
-    compile group: 'com.google.auth', name: 'google-auth-library-oauth2-http', version: '0.16.2'
+    compile group: 'com.google.http-client', name: 'google-http-client', version: '1.39.2'
+    compile group: 'com.google.auth', name: 'google-auth-library-oauth2-http', version: '1.0.0'
     testCompile group: 'org.apache.maven.wagon', name: 'wagon-provider-test', version: '3.3.3'
     testCompile group: 'junit', name: 'junit', version: '4.12'
     compileOnly group: 'org.apache.maven.wagon', name: 'wagon-provider-api', version: '3.3.3'

